extensions/2.0/Vendor/GRIFFEL_bim_data/schema/glTF.GRIFFEL_bim_data.schema.json

```diff
@@ -0,0 +1,58 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "GRIFFEL_bim_data glTF Extension",
+    "type": "object",
+    "description": "Domain specific properties for glTF nodes.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "nodeProperties": {
+            "type": "array",
+            "items": {
+                "$ref": "node.properties.schema.json",
+                "description": "Node properties and type."
+            },
+            "description": "Node type and instance properties map. Used only when data is saved in a separate file."
+        },
+        "propertyValues": {
+            "type": "array",
+            "items": {
+                "type": "string",
+                "description": "Property value.",
+                "minLength": 1
+            },
+            "minItems": 1,
+            "description": "Collection of unique property values."
+        },
+        "propertyNames": {
+            "type": "array",
+            "items": {
+                "type": "string",
+                "description": "Property name.",
+                "minLength": 1
+            },
+            "minItems": 1,
+            "description": "Collection of unique property names."
+        },
+        "properties": {
+            "type": "array",
+            "items": {
+                "$ref": "property.schema.json",
```

Comment:
The `property.schema.json` is quite simple and used only once. so it should be just embedded here.

Reply:
Embedded property.schema.